env_daos

```diff
@@ -1,74 +1,27 @@
-#logs
-#export DD_SUBSYS=ALL
+export FI_MR_CACHE_MAX_COUNT=0
 export D_LOG_FILE=/tmp/daos_client.log
 export D_LOG_MASK=ERR
-#export DD_MASK=all
-
-LOCATION=$1
-
-#var
-daospath=/$LOCATION/install
-daosdeps=/$LOCATION/opt
-CPATH=${daospath}/include/:$CPATH
-CPATH=${daospath}/include/:${daosdeps}/argobots/include:${daosdeps}/cart/include:$CPATH
-CPATH=${daosdeps}/fuse/include:${daosdeps}/hwloc/include:${daosdeps}/isal/include:$CPATH
-CPATH=${daosdeps}/mercury/include:${daosdeps}/mpi4py/include:${daosdeps}/ofi/include:$CPATH
-CPATH=${daosdeps}/ompi/include:${daosdeps}/openpa/include:${daosdeps}/pmdk/include:$CPATH
-CPATH=${daosdeps}/pmix/include:${daosdeps}/spdk/include:$CPATH
-
-#net conf, everything below OFI_PORT is required for psm2, change others as needed
-
 export CRT_PHY_ADDR_STR="ofi+verbs;ofi_rxm"
 export OFI_INTERFACE=ib0
 export OFI_DOMAIN=mlx5_0
+export DAOS_AGENT_DRPC_DIR=/tmp/daos_agent
+export DAOS_DISABLE_REQ_FWD=1
 
-#deps
-ARGOBOTS=${daosdeps}/argobots
-CART=${daosdeps}/cart
-CCI=${daosdeps}/cci
-HWLOC=${daosdeps}/hwloc
-MERCURY=${daosdeps}/mercury
-PMDK=${daosdeps}/pmdk
-OMPI=${daosdeps}/ompi
-OPA=${daosdeps}/openpa
-PMIX=${daosdeps}/pmix
-OFI=${daosdeps}/ofi
-ISAL=${daosdeps}/isal
-SPDK=${daosdeps}/spdk
-PROTOBUF=${daosdeps}/protobufc
+LOCATION=$1
 
-#these two are used by daos_bench
-DAOS_PATH=$daospath
-OMPI_PATH=$OMPI
+daospath=/$LOCATION/install
 
-#alpha/omega
-PATH=$CART/bin/:$CCI/bin/:$HWLOC/bin/:$PATH
-PATH=$MERCURY/bin/:$NVML/bin/:$OMPI/bin/:$OPA/bin/:$PMIX/bin/:$PATH
+PATH=~/.local/bin:$PATH
+PATH=~/mpich-3.3.2/install/bin:$PATH
```

Comment:
Minor: Can we have a bin more generic name for the mpich folder than 'mpich-3.3.2'. Just 'mpich' will do.

How about having mpich under '~/utils/' folder instead of '~/' , which we all already have and keep our 'orterun' and 'hwloc' builds?